app/assets/stylesheets/_authors.scss

```diff
@@ -0,0 +1,9 @@
+.author__name{
```

Comment:
Selector `author__name` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space